### PR TITLE
Do not raise an error when a service is already started

### PR DESCRIPTION
**Pull request by antho1404**
When a service is started and we execute the command `service:start` we have an error.

Now the command will catch the error, display a warning and return the hash based on the error.

### CODE REVIEW

**Comment by antho1404** [comment]
Having these errors is really not developer-friendly. When developing a service we can have this error for example with the dev command, also the workflow that will create and start the service based on the hash or whatever. All this is just blocking the user. That's why I still kept the warning, it's important to know but it doesn't need to crash the task.

And for the part of a script using the cli then, in that case, the script will not crash and you will still have the instance of the service to continue your script. Having an error will sure help but having no error will just remove this problem and you will still be able to handle the real errors like the connection to the engine or stuff like that.

**Comment by krhubert** [comment]
> Having these errors is really not developer-friendly. 

Why so? The errors tells the story and you can move on with developing.

> All this is just blocking the user.

How this is blocking the user, i don't get it :(

> Having an error will sure help but having no error will just remove this problem

On backend side, Script/Task automation doesn't work that way if you remove the error you won't have any pbm. It's rather opposite and my comment was exactly about that. On backend It's more like if you remove the error you will have more pbms. And backend/automation/etc perspectives should also be taken within each PRs. Still I don't see how the PR affects developers, but I see a lot of troubles with setting it up.

File: src/utils/error.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export const isAlreadyExists = (err: Error, resource: string): boolean => {
+  const reg = new RegExp(`${resource} \"(.*)\" already exists`)
+  return reg.test(err.message)
+}
+
+export const resourceHash = (err: Error, resource: string): string => {
+  const reg = new RegExp(`${resource} \"(.*)\" already exists`)
```
Comment (NicolasMahe) [inline src/utils/error.ts]: This should be done using the gRPC Error code... This is not an small fix as the Engine needs to be updated. But using a regex is a hack.

Comment (antho1404) [reply src/utils/error.ts]: definitely agree but unfortunately, for now, the engine doesn't return proper error code.

**Comment by antho1404** [comment]
I did some updates, now the error is still triggered but typed and can be caught by the dev/compile commands

**Pull request merged**